mechanicalsoup/browser.py

```diff
@@ -5,7 +5,7 @@
 from .form import Form
 
 
-class Browser:
+class Browser(object):
```

Comment:
@hickford In Python 2, this is necessary to make `Browser` [a new-style class instead of an old-style class](http://stackoverflow.com/q/54867/744178).  (In Python 3, all classes are new-style, so explicitly inheriting from `object` is superfluous.)